tests/Operation/ChangeStreamFunctionalTest.php

```diff
@@ -129,4 +129,29 @@ public function test_resume_after_kill_then_insert()
         $changeStreamResult->next();
         $this->assertNull($changeStreamResult->current());
     }
+
+    public function test_key()
+    {
+        $operation = new DatabaseCommand("admin", ["setFeatureCompatibilityVersion" => "3.6"]);
+        $operation->execute($this->getPrimaryServer());
+
+        $this->collection = new Collection($this->manager, $this->getDatabaseName(), $this->getCollectionName());
+
+        $changeStreamResult = $this->collection->watch();
+
+        $this->assertSame(0, $changeStreamResult->key());
+
+        $result = $this->collection->insertOne(['x' => 1]);
+        $this->assertInstanceOf('MongoDB\InsertOneResult', $result);
+        $this->assertSame(1, $result->getInsertedCount());
+
+        $changeStreamResult->next();
+        $this->assertSame(1, $changeStreamResult->key());
```

Comment:
If you were to do multiple calls to `$changeStreamResult->next();` here and there were no new changes, would `key()` remain at 1?

Reply:
Acknowledged by changes below.